packages/hardhat-core/test/internal/hardhat-network/helpers/providers.ts

```diff
@@ -67,6 +67,7 @@ export const PROVIDERS = [
     isJsonRpc: false,
     networkId: DEFAULT_NETWORK_ID,
     chainId: DEFAULT_CHAIN_ID,
+    rethnetBinary: undefined as string | undefined,
```

Comment:
I think you can drop this property from the binaries; it's not used anywhere. The rethnet binary is passed to the (inner) `useProvider` call in the definition of the `useProvider` method of the rethnet provider (sorry, I know this is super confusing), so that's enough.

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/4095/commits/cb61fb95c774d3c7204a72d1de250a18d62c03f2